.github/workflows/master_pipeline.yml

```diff
@@ -0,0 +1,55 @@
+name: Main pipeline
+on:
+  workflow_dispatch:
+  push:
+  pull_request:
+    types: [ reopened ]
+
+env:
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_DEFAULT_REGION: eu-central-1
+
+jobs:
+  diff:
+    name: Helmfile Diff
+    runs-on: ubuntu-latest
+    container: "ghcr.io/latin-ua/docker_images/terraform_pipeline:latest"
+    steps: 
+      - name: Checkout source code
+        uses: actions/checkout@v2
+      - name: Run helmfile diff
+        run: |
+          source assume-role.sh
+          aws eks update-kubeconfig --name main-cluster
+          helmfile diff
+
+  add_label:
+    name: Add label and Annotations
+    if: github.ref_name == 'master'
+    runs-on: ubuntu-latest
+    container: "ghcr.io/latin-ua/docker_images/terraform_pipeline:latest"
+    steps: 
+      - name: Checkout source code
+        uses: actions/checkout@v2
+      - name: Run helmfile upgrade
+        run: |
+          source assume-role.sh
+          aws eks update-kubeconfig --name main-cluster
+          kubectl label --overwrite configmaps aws-auth "app.kubernetes.io/managed-by=Helm"
```

Comment:
Please add `-n kube-system` to change the configmap in the appropriate namespace